### PR TITLE
Faq commands

### PR DESCRIPTION
**Pull request by DisNin**
1. Add faq commands for once modules
2. Add wikipedia command
3. Add command which sends a link to the kami-blue repositories

uwu

### CODE REVIEW

**Comment by humboldt123** [comment]
This is a giant pull request. Give me some time to review.

**Comment by DisNin** [comment]
> This is a giant pull request. Give me some time to review.

np

File: commands/faq.githublink.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+module.exports.run = async (client, message, args) => {
+    if ((message.channel.id == "724349410858893372") || (message.channel.id == "722436626248237076")) {
+        let githublinkRU = new Discord.MessageEmbed()
+        .setTitle(`[ Оффициальные репозитории KAMI Blue ](https://github.com/kami-blue)`)
+        .setColor(client.colors.kamiblue)
+        message.channel.send(githublinkRU)
+    } else {
+        let githublinkEN = new Discord.MessageEmbed()
+        .setTitle(`[ Official repositories KAMI Blue ](https://github.com/kami-blue)`)
```
Comment (humboldt123) [inline commands/faq.githublink.js:12]: Remove the whitespace, change it to this
```js
        .setTitle(`[Official repositories KAMI Blue](https://github.com/kami-blue)`)
```

**Review by humboldt123 — CHANGES_REQUESTED** [review]

**Comment by dominikaaaa** [comment]
I'd prefer the module faq commands would be in 1 large command with aliases or subcommands

File: commands/faq.githublink.js (hunk 1 above)
Comment (DisNin) [reply commands/faq.githublink.js]: done

**Comment by DisNin** [comment]
> I'd prefer the module faq commands would be in 1 large command with aliases or subcommands

i am new to js, ​​just started learning arrays lol

**Comment by DisNin** [comment]
lmao 
Screenshot_9

**Comment by DisNin** [comment]
maybe just make page 0 and in one of the commands do `use:" faq / "command_name" `

**Comment by DisNin** [comment]
bruh

**Comment by DisNin** [comment]
mm, okay now it should be work nice

**Comment by DisNin** [comment]
@humboldt123 = ^

**Comment by sourTaste000** [comment]
why bot-js....

**Comment by sourTaste000** [comment]
im porting this to kotlin

**Comment by sourTaste000** [comment]
wait why the fuck this is hardcoded
just use json..

**Comment by DisNin** [comment]
based

**Comment by DisNin** [comment]
yes, im rewrite it soon, because im learning

**Pull request merged**